_pages/coc.md

```diff
@@ -74,7 +80,17 @@ communications pertaining to community business.
 
 ## Contact Us ##
```

Comment:
change this to "how to report a violation"?

Reply:
ehh it's linked to in a couple of other places and one is reporting a grievance which you would still use these things for but it wouldn't be for reporting a violation

are you ok keeping as is?